<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1070</width>
    <height>836</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="verticalWidget" native="true">
     <property name="styleSheet">
      <string notr="true">background-color:#283141;color:white;</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>9</number>
      </property>
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="styleSheet">
         <string notr="true">border:none</string>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>1052</width>
           <height>818</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>40</number>
            </property>
            <property name="rightMargin">
             <number>6</number>
            </property>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="font">
               <font>
                <pointsize>21</pointsize>
               </font>
              </property>
              <property name="text">
               <string>心率：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="xllineEdit">
              <property name="minimumSize">
               <size>
                <width>300</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>300</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>21</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color:#354052;border:none;</string>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="font">
               <font>
                <pointsize>21</pointsize>
               </font>
              </property>
              <property name="text">
               <string>心律：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="xnlineEdit">
              <property name="maximumSize">
               <size>
                <width>300</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>21</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color:#354052;border:none;</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <property name="leftMargin">
             <number>40</number>
            </property>
            <property name="rightMargin">
             <number>6</number>
            </property>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="font">
               <font>
                <pointsize>21</pointsize>
               </font>
              </property>
              <property name="text">
               <string>RR：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="RRLineEdit">
              <property name="maximumSize">
               <size>
                <width>300</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>21</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color:#354052;border:none;</string>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_6">
              <property name="font">
               <font>
                <pointsize>21</pointsize>
               </font>
              </property>
              <property name="text">
               <string>收缩压：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="souLineEdit">
              <property name="maximumSize">
               <size>
                <width>300</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>21</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color:#354052;border:none;</string>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="leftMargin">
             <number>40</number>
            </property>
            <property name="rightMargin">
             <number>6</number>
            </property>
            <item>
             <widget class="QLabel" name="label_7">
              <property name="font">
               <font>
                <pointsize>21</pointsize>
               </font>
              </property>
              <property name="text">
               <string>舒张压：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="suLineEdit">
              <property name="maximumSize">
               <size>
                <width>300</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>21</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color:#354052;border:none;</string>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_8">
              <property name="font">
               <font>
                <pointsize>21</pointsize>
               </font>
              </property>
              <property name="text">
               <string>MAP：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="mapLineEdit">
              <property name="maximumSize">
               <size>
                <width>300</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>21</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color:#354052;border:none;</string>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <property name="leftMargin">
             <number>40</number>
            </property>
            <property name="rightMargin">
             <number>6</number>
            </property>
            <item>
             <widget class="QLabel" name="label">
              <property name="font">
               <font>
                <pointsize>21</pointsize>
               </font>
              </property>
              <property name="text">
               <string>PI：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="PILineEdit">
              <property name="maximumSize">
               <size>
                <width>300</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>21</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color:#354052;border:none;</string>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="font">
               <font>
                <pointsize>21</pointsize>
               </font>
              </property>
              <property name="text">
               <string>SPO2：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="SPOlineEdit">
              <property name="maximumSize">
               <size>
                <width>300</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>21</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color:#354052;border:none;</string>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="leftMargin">
             <number>40</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>6</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="UsCodelabel">
              <property name="font">
               <font>
                <pointsize>21</pointsize>
               </font>
              </property>
              <property name="text">
               <string>超声编号：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="USCOdelineEdit">
              <property name="maximumSize">
               <size>
                <width>800</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>21</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color:#354052;border:none;</string>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
